src/System.Runtime.Intrinsics/src/System/Runtime/Intrinsics/Types.cs

```diff
@@ -0,0 +1,18 @@
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Intrinsics
+{
+    // 128 bit types
+    [StructLayout(LayoutKind.Sequential, Size = 16)]
+    public struct Vector128<T> where T : struct 
```

Comment:
If these APIs will be implemented in S.P.CoreLib, you do not need the files under src. You just need the files under ref.